### DISCUSSION TITLE
Windows version 22623.1095

### DISCUSSION THREAD

**Discussion by Staleyy** [PeterStrick/ViVeTool-GUI#44] [Q&A]
Im kinda annoyed that there is no version 22623.1095 to edit in ViveTool
how do i enable it/ use it then?

**Comment by PeterStrick** [comment]
There are currently no Features Documented / no Feature List exists for Build 22623.*

You could try hunting down ViVeTool IDs for your Build and manually enabling them in ViVeTool GUI by pressing F12
